### PR TITLE
feat: add issue comment TFIDF similarity metrics and issue comment Ja…

### PR DESCRIPTION
**Pull request by bifenglin**
This PR adds the metric about actors' comment TFIDF similarity and actors' Jaccard similarity.

### CODE REVIEW

**Comment by xgdyp** [comment]
hi @bifenglin , if this pr has not completed, you can convert it to draft PR

File: python/metrics/index.py [hunk 1]
```diff
@@ -15,6 +16,10 @@ def getRelatedUsers(config):
     return related_users.getRelatedUsers(config)
 def getAttention(config):
     return attention.getAttention(config)
+def getIssueCommentTFIDFSimilarity(config):
```
Comment (xgdyp) [inline python/metrics/index.py:19]: It seems that `config` has never been used

File: python/metrics/issue_comment_similartiy.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+from nltk.corpus import stopwords
+from sklearn.feature_extraction.text import CountVectorizer
+from scipy.linalg import norm
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+def getSelectedActors(config):
+    """
+    TODO: get Selected Acotrs
+    """
+    sql = 'SELECT DISTINCT(actor_id) FROM opensource.gh_events')
```
Comment (xgdyp) [inline python/metrics/issue_comment_similartiy.py:15]: I think we can add a limit clause such as `limit 10`

Comment (bifenglin) [reply python/metrics/issue_comment_similartiy.py:15]: The function aims to find all users if have no selecting choice.

Comment (frank-zsy) [reply python/metrics/issue_comment_similartiy.py:15]: `DISTINCT` selection from the whole table may lead to a OOM exception since the ClickHouse only has 128GB memory, we should definitely avoid this kind of SQL.

**Comment by xgdyp** [comment]
hi, I think this pr can be converted into a case in  `notebook`. I'll rewrite it.

**Comment by lhbvvvvv** [comment]
hi @bifenglin. I'm adding this to the `notebook` recently.  I think there is something to be confirmed below  :

- Both the code and the pr description are about the similarity of actors, so the pr title should be changed accordingly.
- `getRecent100Comment` in `getIssueCommentTFIDFSimilarity` is not defined .
- In `config`, do we want to pass the actors we want to query into the function?
